core/templates/dev/head/css/oppia.css

```diff
@@ -3657,7 +3657,7 @@ md-card.preview-conversation-skin-supplemental-card {
 .oppia-about-header {
   background-blend-mode: multiply;
   background-color: #afd2eb;
-  background-image: url("../images/general/icons-bg.png");
+  background-image: url("../assets/common/images/general/icons-bg.png");
```

Comment:
I had thought that it might be used in a nested page, but that's not true. It's used in about.html which is just one step down. So, we could safely remove `..`.